packages/kit/src/core/postbuild/prerender.js

```diff
@@ -341,7 +341,7 @@ async function prerender({ out, manifest_path, metadata, verbose, env }) {
 			if (location) {
 				const resolved = resolve(encoded, location);
 				if (is_root_relative(resolved)) {
-					enqueue(decoded, decode_uri(resolved), resolved);
+					await enqueue(decoded, decode_uri(resolved), resolved);
```

Comment:
why are we awaiting these all? I imagine this adding up when prerendering a lot of files, hurting perf in the end, and I can't really imagine there being errors that are otherwise uncaught.

Reply:
it's such a quick action I didn't really imagine it adding up. I've updated it to enqueue them in parallel instead though to be safe

Reply:
See, stuff like this is exactly why these rules are so dangerous — I'm pretty sure this creates deadlock, and I'm pretty sure that's why the tests started failing with the commit that added this.

You have to treat ESLint like an idiot child. You can give it very simple tasks from time to time, but don't trust it with anything important.

Let's revert all this